src/block/CustomiesBlockFactory.php

```diff
@@ -42,8 +46,10 @@ final class CustomiesBlockFactory {
 	 * @phpstan-var array<string, Block>
 	 */
 	private array $customBlocks = [];
+
```

Comment:
You still unnecessary formatting in multiple files, like this one.

Reply:
Sorry my IDE did it automatically and I must not have seen it when sending the commit